dlc2nwb/utils.py

```diff
@@ -51,36 +51,11 @@ def _ensure_individuals_in_header(df, dummy_name):
     return df
 
 
-def convert_h5_to_nwb(config, h5file, individual_name="ind1"):
-    """
-    Convert a DeepLabCut (DLC) video prediction, h5 data file to Neurodata Without Borders (NWB). Also
-    takes project config, to store relevant metadata.
-
-    Parameters
-    ----------
-    config : str
-        Path to a project config.yaml file
-
-    h5file : str
-        Path to a h5 data file
-
-    individual_name : str
-        Name of the subject (whose pose is predicted) for single-animal DLC project.
-        For multi-animal projects, the names from the DLC project will be used directly.
-
-    TODO: allow one to overwrite those names, with a mapping?
-
-    Returns
-    -------
-    list of str
-        List of paths to the newly created NWB data files.
-        By default NWB files are stored in the same folder as the h5file.
-
-    """
+def _get_pes_args(h5file, individual_name, config_file):
     if "DLC" not in h5file or not h5file.endswith(".h5"):
         raise IOError("The file passed in is not a DeepLabCut h5 data file.")
 
-    cfg = auxiliaryfunctions.read_config(config)
+    cfg = auxiliaryfunctions.read_config(config_file)
 
     vidname, scorer = os.path.split(h5file)[-1].split("DLC")
```

Comment:
what if the h5file has multiple individuals' info( like on [line 199](https://github.com/DeepLabCut/DLC2NWB/pull/10/files#diff-93850a833725a5cac865f8a3422f61c58aebbe09cc7f1cacf546924632d84ce1R199), with each individual having one video from which pose is estimated. How will the `h5file` be named in such a case?

Reply:
In DeepLabCut, h5 files will be named the same, regardless of whether they contain multiple animals from a video, or a distinct individual per video ☺️